lib/Adapter/WorseReflection/Transformer/AddMissingProperties.php

```diff
@@ -44,7 +44,8 @@ public function transform(SourceCode $code): SourceCode
 
         /** @var ReflectionClass $class */
         foreach ($classes as $class) {
-            $classBuilder = $sourceBuilder->class($class->name()->short());
+            $classOrTrait = $class->isTrait() ? 'trait' : 'class';
+            $classBuilder = $sourceBuilder->$classOrTrait($class->name()->short());
```

Comment:
Can we avoid dynamic method names? e.g. `$classBuilder = $class->isTrait() ? $sourceBuiler->trait() : $sourceBuilder->class(); ...`